spyder/api/plugins.py

```diff
@@ -528,8 +796,159 @@ def toggle_view(self, checked):
         else:
             self.dockwidget.hide()
 
+    # FIXME: What are some examples of this usage?
```

Comment:
Please look for them. @dalthviz added this method.

Reply:
@dalthviz could you please point to why we need this? seems like we could most likely define this automatically? I may be wrong so please explain :-)